src/UserSpaceInstrumentation/TrampolineTest.cpp

```diff
@@ -651,10 +671,10 @@ extern "C" int DoSomething() {
 
 TEST_F(InstrumentFunctionTest, DoSomething) {
   RunChild(&DoSomething, "DoSomething");
-  PrepareInstrumentation("TrivialLog");
-  ErrorMessageOr<uint64_t> address_after_prologue_or_error =
-      CreateTrampoline(pid_, function_address_, function_code_, trampoline_address_,
-                       payload_function_address_, capstone_handle_, relocation_map_);
+  PrepareInstrumentation("EntryPayload", "ExitPayload");
```

Comment:
I suggest creating constants for these two strings.

Reply:
done